packages/superset-ui-style/src/index.ts

```diff
@@ -111,6 +117,12 @@ const defaultTheme = {
       xxl: 28,
     },
   },
+  zIndex: {
+    aboveDashboardCharts: 10,
+    dropdown: 11,
+    max: 3000,
+  },
+  timingNormal: 0.3,
```

Comment:
Maybe this could have a more descriptive name? I'm not sure what `timingNormal` means.

Reply:
If this is use for CSS animation, maybe use something like `transitionDurationDefault`.

Reply:
named it `transitionTiming`